common/tools/dev-tool/src/util/customization/aliases.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import { SourceFile, TypeAliasDeclaration } from "ts-morph";
+
+export function augmentTypeAliases(
+  originalAliases: Map<string, TypeAliasDeclaration>,
+  customAliases: TypeAliasDeclaration[],
+  originalFile: SourceFile
+) {
+  for (const customInterface of customAliases) {
+    const originalAlias = originalAliases.get(customInterface.getName() ?? "");
```

Comment:
Probably not gonna happen in practice but should we throw an error in case the interface doesn't have a name?

Reply:
Good idea!